doc/welcome.en.tm

```diff
@@ -7,8 +7,8 @@
 
   <with|color|dark red|<\warning*>
     Please use <hlink|<name|Mogan Editor>
-    v1.1.0|https://github.com/XmacsLabs/mogan/releases/tag/v1.1.0> if you are
-    viewing this document with messy code (They should be Chinese
+    v1.1.0|https://github.com/XmacsLabs/mogan/releases/tag/v1.1.0> or later
```

Comment:
v1.1.1

Always suggest user to download the latest version. Or just remove the link here.